internal/client/client.go

```diff
@@ -100,30 +105,33 @@ func Run(addr, fingerprint, proxyAddr string) {
 
 	sshPriv, sysinfoError := keys.GetPrivateKey()
 	if sysinfoError != nil {
-		log.Fatal("Getting private key failed: ", sysinfoError)
+		log.Println("Getting private key failed: ", sysinfoError)
 	}
 
 	l := logger.NewLog("client")
 
-	var username string
-	userInfo, sysinfoError := user.Current()
-	if sysinfoError != nil {
-		l.Warning("Couldnt get username: %s", sysinfoError.Error())
-		username = "Unknown"
-	} else {
-		username = userInfo.Username
+	if username == "" {
+		userInfo, sysinfoError := user.Current()
+		if sysinfoError != nil {
+			l.Warning("Couldnt get username: %s", sysinfoError.Error())
+		} else {
+			username = userInfo.Username
+		}
 	}
 
-	hostname, sysinfoError := os.Hostname()
-	if sysinfoError != nil {
-		hostname = "Unknown Hostname"
-		l.Warning("Couldnt get host name: %s", sysinfoError)
-	}
+	/*
+		hostname, sysinfoError := os.Hostname()
+		if sysinfoError != nil {
+			hostname = "Unknown Hostname"
+			l.Warning("Couldnt get host name: %s", sysinfoError)
+		}
+	*/
 
 	config := &ssh.ClientConfig{
-		User: fmt.Sprintf("%s.%s", username, hostname),
+		User: username, //fmt.Sprintf("%s.%s", username, hostname),
```

Comment:
Omits important information

Reply:
Which user started it? In fact, once we are connected, there are many commands that can be obtained? So actually it's not that important here?

Reply:
It omits the hostname, and the real user that the rssh binary is running as